README.md

```diff
@@ -42,34 +42,44 @@ implementation 'com.github.imagekit-developer:imagekit-android:<VERSION>'
 
```

Comment:
Add a table of index and link all important sections of this readme at the top like this - https://github.com/imagekit-developer/imagekit-nodejs#table-of-contents